src/character.cpp

```diff
@@ -1011,7 +1011,8 @@ int Character::swim_speed() const
 
 bool Character::is_on_ground() const
 {
-    return get_working_leg_count() < 2 || has_effect( effect_downed );
+    return ( ( get_working_leg_count() < 2 && !weapon.has_flag( flag_CRUTCHES ) ) ) ||
```

Comment:
Will the player fall over when trying to attack with their crutches?

Reply:
I did not really thought of that. Should he fall over but still deal damage or just fall over and that's it?

Reply:
Maybe we can make it depend on melee skill, or maybe we can just display a message that says you can't attack with crutches when your legs are broken.

Reply:
I've added check when performing melee if legs are broken. Currently you just cannot melee attack while you have broken legs. I don't think melee skill should affect that. But that's just a balance issue